raspiaudio-muse-luxe.yaml

```
@@ -108,22 +166,62 @@ voice_assistant:
 sensor:
   - platform: adc
     pin: GPIO33
-    name: Battery
-    icon: "mdi:battery-outline"
+    name: Battery voltage
     device_class: voltage
+    unit_of_measurement: "V"
```

Don't need unnecessary quotes
```suggestion
    unit_of_measurement: V
```